Core/TImeSeries/TimeSeries.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Core;
+
+namespace Core.TimeSeries
+{
+	public class TimeSeries
+	{
+		protected Date End = Date.MinValue;
+
+		protected Date Start = Date.MaxValue;
+
+		protected List<double> Series { get; } = new List<double>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this property to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)